thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java

```diff
@@ -128,12 +149,15 @@ public void buildReport(long startTime, long endTime, List<MergedAnomalyResultDT
       templateData.put("nonActionableCount", nonActionable);
       templateData.put("anomalyDetails", anomalyReportDTOList);
       buildEmailTemplateAndSendAlert(templateData, configuration.getSmtpConfiguration(),
-          emailRecipients);
+          emailRecipients, fromEmail);
     }
   }
 
   void buildEmailTemplateAndSendAlert(Map<String, Object> paramMap,
-      SmtpConfiguration smtpConfiguration, String emailRecipients) {
+      SmtpConfiguration smtpConfiguration, String emailRecipients, String fromEmail) {
+    if (Strings.isNullOrEmpty(fromEmail)) {
+      fromEmail = "thirdeye-dev@linkedin.com";
```

Comment:
Could not find better suffix, but def. can remove it once we move completely to this.
I will remove the from email hardcoding.